### PR TITLE
[docker] Change default OMP_NUM_THREADS back to 1 for GPU

### PR DESCRIPTION
**Pull request by frankfliu**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Review by lanking520 — COMMENTED** [review]
Where do you set to 1?

**Comment by frankfliu** [comment]
It's set in the startup shell script: https://github.com/deepjavalibrary/djl-serving/blob/master/serving/build.gradle#L109-L110

**Comment by lanking520** [comment]
OK, then you cannot enable OMP_TREAD=1 for INF2, it will slow down SD inference speed.

**Comment by frankfliu** [comment]
The problem is what should be the default value.

without OMP_NUM_THREADS=1:
1. most of CV models will be suffered
2. LLM is not impacted by OMP_NUM_THREADS
3. SD model has sighttly better performance

I think we should document this for SD model and use OMP_NUM_THREADS=1 by default

**Comment by lanking520** [comment]
I would suggest you to set all other containers to OMP_THREAD=1 besides Neuron container.

File: serving/docker/pytorch-inf2.Dockerfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ COPY dockerd-entrypoint.sh /usr/local/bin/dockerd-entrypoint.sh
 RUN chmod +x /usr/local/bin/dockerd-entrypoint.sh
 WORKDIR /opt/djl
 ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
-ENV NO_OMP_NUM_THREADS=true
+# ENV NO_OMP_NUM_THREADS=true
```
Comment (lanking520) [inline serving/docker/pytorch-inf2.Dockerfile:33]: let's keep INF2 change as is. And rest looks all good

**Review by lanking520 — APPROVED** [review]

**Pull request merged**